### ISSUE TITLE
BUG: Gradient normalization in SPSA optimizer breaks algorithm

### ISSUE THREAD

**Issue by AntonJoha** [AntonJoha/SPSA-pytorch#41]
|
